client/my-sites/plans/current-plan/index.jsx

```diff
@@ -95,13 +96,25 @@ class CurrentPlan extends Component {
 		};
 	}
 
+	getThankYou() {
```

Comment:
Can we rename this? `get()` implies that we're getting or computing some data but what this really does is render something. 

```suggestion
	renderThankYou() {
```